index.html

```diff
@@ -21,174 +28,217 @@
           </a>
         </nav>
       </div>
-      <div class="intro">
-        <h1 class="name">Pyonguk<br>Agustin</h1>
-        <div class="paragraph">
-          <p>
-            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
-          </p>
+      <div id="dt_intro">
+        <div class="intro">
+          <h1 class="name">Pyonguk<br>Agustin</h1>
+          <div class="paragraph">
+            <p>
+              Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+          </div>
+          <a class="btn" href="#"><h6>Start Collaboration</h6></a>
+          <img id="dots1" src="icons/dots_grid.svg" alt="dots">
+        </div>
+        <div id="dt_geom_intro">
+          <img src="icons/dektop_geometry_intro.svg" alt="geometry">
         </div>
-        <a class="btn" href="#">Start Collaboration</a>
       </div>
     </header>
     <div class="geometrical_div">
       <img src="icons/geometry.png" alt="A geometrical division">
     </div>
-    <h1 class="main_title">Projects</h1>
     <main>
-      <section class="project">
-        <div class="template">
-          <img src="icons/project1.svg" alt="Example image">
-        </div>
-        <div class="descr">
-          <h2>Project name goes here</h2>
-          <div class="lang">
-            <h3>HTML/CSS</h3>
-            <h3>Ruby on Rails</h3>
-            <h3>JavaScript</h3>
-          </div>
-          <button type="button">See This Project →</button>
```

Comment:
- [ ] [OPTIONAL] Can you please try to decrease the font-size of this button because it seems to be bigger than the one in Figma design 👍